pages/jobs/index.js

```diff
@@ -33,21 +45,36 @@ export default function JobsIndexPage() {
              Job Postings
             </h2>
             <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
-              Want to work on a great open-source project with a lovely team and a vibrant community? Come work with us!
+              Want to work on a great open-source project with a lovely team and a vibrant community? Browse through available job offers!
+            </p>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
+            <span role="img">💡</span> Want to post a job offer? <a className="ml-1 text-primary-500 hover:text-primary-400" href={`https://github.com/asyncapi/website/new/master/pages/jobs?value=${jobPostUrl}`} target="_blank">Post it now!</a>
+            </p>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400"> Do you want to discuss your job offer first?
+            <a className="ml-1 text-primary-500 hover:text-primary-400" href="https://github.com/asyncapi/website/issues/new" target="_blank">Get started here.</a>
             </p>
           </div>
-          <div className="mt-8 flex flex-col items-stretch sm:rounded-md">
-            <ul className="bg-white shadow overflow-hidden divide-y divide-gray-200 sm:w-2/3 sm:self-center">
-            {
-              posts.map((post, index) => (
-                <JobPostItem key={index} job={post} />
-              ))
-            }
-            </ul>
+          <div className="text-center">
+            {emptyChecker(posts) ? <div className="flex content-center justify-center">
```

Comment:
What about this?

```suggestion
            {!Object.keys(posts).length ? <div className="flex content-center justify-center">
```

Reply:
Wow!!! That's cool, then i don't need the empty checker helper.

Reply:
But do you think it's better than the empty checker helper function?

Reply:
At least it's self-contained. I mean, it doesn't need to import something on another file. If you want to make it easier to understand you can probably define a variable before the return like this:

```js
const hasPosts = Object.keys(posts).length;
```

Then you can use it like `{!hasPosts ? <div...`

Reply:
Ok cool

Reply:
@fmvilas I've updated the PR, please kindly review